src/intro-view/intro-view.html

```diff
@@ -10,6 +10,10 @@
       :host {
         display: block;
       }
+      img {
+        max-width: 80%;
+        max-height: 80%;
```

Comment:
Any reason it's not just width and height? Why just max? I know I prefer to lay out dynamic sizes with proportions.